Tools/TestWebKitAPI/Tests/WTF/URLParser.cpp

```diff
@@ -944,8 +944,13 @@ TEST_F(WTF_URLParser, ParserDifferences)
     checkURL(utf16String(u"http://ԛәлп.com"), { "http"_s, ""_s, ""_s, "xn--k1ai47bhi.com"_s, 0, "/"_s, ""_s, ""_s, "http://xn--k1ai47bhi.com/"_s });
     checkURLDifferences(utf16String(u"http://Ⱥbby.com"),
         { "http"_s, ""_s, ""_s, "xn--bby-iy0b.com"_s, 0, "/"_s, ""_s, ""_s, "http://xn--bby-iy0b.com/"_s });
+#if U_ICU_VERSION_MAJOR_NUM >= 76
```

Comment:
Can we check it with WTF's ICU version function? OSS WebKit on macOS is built with the oldest supported ICU headers. Thus, ICU header version does not match against the running ICU version. So runtime check is necessary.

Reply:
I didn't know it. Fixed. Thank you.